src/translations/ref.json

```diff
@@ -1,5 +1,5 @@
 {
-  "appName": "openIMIS",
+  "appName": "CORE-MIS",
```

Comment:
@sniedzielski @delcroip is this a configuration option somewhere? I don't see it in https://openimis.atlassian.net/wiki/spaces/OP/pages/4155342851/Configuration+items+for+specific+openIMIS+solutions

Reply:
No. That could be worth to add this. Thanks for pointing this.